scripts/AbstractWidget.py

```diff
@@ -0,0 +1,322 @@
+import re
+import sys
+import os
+class eventHandlerInfo:
```

Comment:
nit: typically class name is like `EventHandlerInfo`, and module name is e.g. `abstract_widget.py`